### PR TITLE
feat(JOML): migrate aabb for SkeletonRenderer

### PR DESCRIPTION
**Pull request by pollend**

### CODE REVIEW

File: engine/src/main/java/org/terasology/rendering/gltf/GLTFAnimationFormat.java [hunk 1]
```diff
@@ -159,7 +160,7 @@ private MeshAnimationData loadAnimation(GLTF gltf, GLTFAnimation animation, List
             frames.add(frame);
         }
 
-        return new MeshAnimationData(boneNames, boneParents, frames, TIME_PER_FRAME, AABB.createEmpty());
+        return new MeshAnimationData(boneNames, boneParents, frames, TIME_PER_FRAME, new AABBf());
```
Comment (DarkWeird) [inline engine/src/main/java/org/terasology/rendering/gltf/GLTFAnimationFormat.java:163]: `new AABBf()` creates with (minX, minY, minZ)=(+inf, +inf, +inf) and (maxX, maxY, maxZ)=(-inf, -inf, -inf)

**Review by DarkWeird — CHANGES_REQUESTED** [review]

**Review by jdrueckert — CHANGES_REQUESTED** [review]
This PR seems to break the model rendering.
On current develop I was able to see both deer and gooey citizens in MR, but with this PR checked out the models were invisible.
image

**Comment by DarkWeird** [comment]
> This PR seems to break the model rendering.
> On current develop I was able to see both deer and gooey citizens in MR, but with this PR checked out the models were invisible.
> image

I think it related to my review. try to find model at infinity :D

File: engine/src/main/java/org/terasology/rendering/gltf/GLTFAnimationFormat.java [hunk 2]
```diff
@@ -159,7 +160,7 @@ private MeshAnimationData loadAnimation(GLTF gltf, GLTFAnimation animation, List
             frames.add(frame);
         }
 
-        return new MeshAnimationData(boneNames, boneParents, frames, TIME_PER_FRAME, AABB.createEmpty());
+        return new MeshAnimationData(boneNames, boneParents, frames, TIME_PER_FRAME, new AABBf(0, 0, 0));
```
Comment (skaldarnar) [inline engine/src/main/java/org/terasology/rendering/gltf/GLTFAnimationFormat.java:163]: Thanks @DarkWeird , this indeed fixed it.

**Review by skaldarnar — APPROVED** [review]

**Review by DarkWeird — APPROVED** [review]

**Pull request merged**